src/main/resources/db/migration/common/V1_259__retention_policies.sql

```diff
@@ -0,0 +1,25 @@
+ALTER TABLE case_management_retention ADD COLUMN is_manual_override   BOOLEAN NOT NULL;
```

Comment:
not sure if its needed, as we may not have any data in the tables, but if introducing a "NOT NULL" column to an existing table and be careful and maybe add it as nullable, populate it with default values, then add the NOT NULL part.